README.md

```diff
@@ -12,32 +12,32 @@ Group 11 project for SEG 2900 at uOttawa
+
+Codes Comp is a website that measures users' coding skills as well as contributes to their coding skills. Players improve their coding skills and develop algorithms by competing with their opponents. Thanks to the Code Comp website, instructions are given to the person's computer to indicate how to solve a particular problem. Therefore, while coding for the given problem, it is necessary to think about how to follow a path to solve it and what steps to take first. If an incorrect command is entered into the computer during coding, the desired result will not be achieved, and different methods will have to be tried to solve the problem. When solving problems in code snippets used, there is a need to try different approaches, so after a while, experience in problem-solving will be gained and reflections on real life. 
+
+During the development of the given code, when it sees an illogical or inappropriate code snippet for the desired syntax, it will immediately give an error. This situation causes the need for alternative thinking to find the source of the error and multiple analyses from different approaches. During these operations, a critical approach to events and the further development of this skill are provided. The problem-solving approach found by the user himself will also increase the person's self-confidence when it gives the same result as other solutions. It is encouraged to try different solutions to reach the result, approach events from different perspectives, and increase creativity.Through this competition in the game, players compete with each other under the same goal, thus causing them to improve themselves. Competitors who want to achieve the goal during the competition focus only on their goals. Focus, a source of motivation, increases the chances of success.
```

Comment:
This looks a lot better! I think that shortening it and also including more information on the product itself rather than the reason we are creating would be beneficial.